### PR TITLE
Login接口

### PR DESCRIPTION
**Pull request by xianxian286**

### CODE REVIEW

File: src/main/java/cn/msa/msa_museum_server/entity/UserEntity.java
Comment (kroscedu) [inline src/main/java/cn/msa/msa_museum_server/entity/UserEntity.java:1]: Useless member [email].

**Review by kroscedu — CHANGES_REQUESTED** [review]
❌

**Review by RayWu-1 — APPROVED** [review]
Approved

File: src/main/java/cn/msa/msa_museum_server/MsaMuseumServerApplication.java
Comment (fuelatank) [inline src/main/java/cn/msa/msa_museum_server/MsaMuseumServerApplication.java:1]: This is not a controller?

**Review by kroscedu — APPROVED** [review]
Wait to be modified.

**Review by wwzzhhrr — COMMENTED** [review]
approve

**Review by winter7zowo — APPROVED** [review]
# nihao

**Review by wwzzhhrr — APPROVED** [review]
ok

File: src/main/java/cn/msa/msa_museum_server/controller/UserController.java
Comment (fuelatank) [inline src/main/java/cn/msa/msa_museum_server/controller/UserController.java:1]: `userRepository` seems to be unused and should not be directly linked to a controller
tab
`RequestBody` is not required in a `RestController`

**Comment by Hikariks** [comment]
k

**Pull request merged**